foundation-extensions/experience-platform-extension/edge-network-api-reference.md

```diff
@@ -178,34 +196,47 @@ MobileCore.registerExtensions([Edge.self, ...], {
 
 Use the AEPMobileCore API to register the Edge extension.
 
-#### Syntax
+**Syntax**
 
-```text
-// Swift MobileCore
-public static func registerExtensions(_ extensions: [Extension.Type], _ completion: (() -> Void)? = nil)
+```objectivec
++ (void) registerExtensions: (NSArray<Class*>* _Nonnull) extensions 
+                  completion: (void (^ _Nullable)(void)) completion;
 ```
 
-#### Example
+**Example**
+
+```objectivec
+@import AEPEdge;
+
+[AEPMobileCore registerExtensions:@[AEPMobileEdge.class] completion:nil];...
 
-```text
-[AEPMobileCore registerExtensions:@[AEPMobileEdge.class, ...] completion:^{
-  // processing after registration
-}];
 ```
+
 {% endtab %}
 {% endtabs %}
 
+## resetIdentities
+
+Clears all the states stored in the Edge extension.
+
+See [MobileCore.resetIdentities](../mobile-core/mobile-core-api-reference.md#resetidentities) for more details.
+
 ## Public classes
 
+### Schema interface
```

Comment:
Rearranging suggestions:
- Let's rename this header to XDM Schema
- Include the following description above the tabs: 
```
The AEP Edge extension provides the Schema interface (Android) / XDMSchema protocol (iOS) that can be used to define the classes associated with your XDM schema in Adobe Experience Platform.
When defining your custom XDM schema(s), implement these interfaces to ensure that the AEP Edge extension successfully serializes the provided data before sending it to Adobe Experience Platform Edge Network.
```
- Then remove this common description from the tabs.

Reply:
This sentence "When defining your custom XDM schema(s), implement these interfaces to ensure that the AEP Edge extension successfully serializes the provided data before sending it to Adobe Experience Platform Edge Network." seems like applying to Java only?  I left it under property interface in Java section as before.  

Please check this section see if the rearrangement is ok.